certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -12334,17 +12334,20 @@
         "representingOrganizationRecordNumber": {
           "type": "string",
           "description": "Representing Organization Record Number.",
-          "title": "Representing Organization Record Number"
+          "title": "Representing Org Record Number",
```

Comment:
@RjImpx please make sure everything is grammatically correct and is free of typographical errors, title and description should not be copy and paste. You can keep the previous full title and make a bit more detailed description. You can refer to [English strings guideline](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) if needed.

We'll be good to go after the fix, thank you!